src/App.tsx

```diff
@@ -1,10 +1,74 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+// eslint-disable-next-line import/no-cycle
+import { Footer } from './components/Footer';
+// eslint-disable-next-line import/no-cycle
+import { ErrorNotification } from './components/ErrorNotification';
+
+const USER_ID = 10380;
+
+export enum Filter {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+export enum ErrorType {
+  Get = 'get',
+  Post = 'post',
+  Delete = 'delete',
+  Patch = 'patch',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
```

Comment:
My assumption is better use empty array as default value
```suggestion
  const [todos, setTodos] = useState<Todo[]>([]);
```